### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by chriszs**
This adds a Dockerfile and .dockerignore file to allow the project to be built as a docker image. This is the first of several incremental changes for Docker readiness outlined in #54.

## Due Diligence Checklist
- [x] Consolidated commits

## Steps to Test
1. `docker build .`
2. `docker run --env-file=.env <image hash>`

## Related Issues
Begins work outlined in #54

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+COPY package.json ./
+COPY yarn.lock ./
+
+RUN yarn
+
+COPY . .
+
+RUN apk add  --no-cache ffmpeg
+
+CMD [ "npm", "start" ]
```
Comment (slifty) [inline Dockerfile:14]: Should we have it run kafka too? or were you thinking kafka would be loaded in a separate container and docker-composed together?

Comment (chriszs) [reply Dockerfile:14]: Oh, separate container! Docker Composed together.

Comment (chriszs) [reply Dockerfile:14]: Modifying Docker Compose is step 3 in my multi-step plan.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM node:14-alpine
```
Comment (reefdog) [inline Dockerfile:1]: The answer to this question is probably tied up in #50, but is it okay if this drifts from the version specified in our `package.json` (currently 13)? I have no particular Node version allegiance, but a slight preference for LTS > current (unless we have specific features we need that aren't in LTS). Mostly just curious if we need to ensure this is in sync with the package file.

Comment (chriszs) [reply Dockerfile:1]: `package.json` specifies a minimum, so I think exceeding it is okay. But Travis is also set to test on 13, so there are a couple signals 13 is the current default. It's also EOL now, which we mostly shouldn't care about, except that we want the Docker image and everything else to have the longest functional life possible.

**Comment by slifty** [comment]
Sanity check: given some of the discussion in #66 should we hold off on merging this? (if not, I think we're probably OK to merge)

**Comment by chriszs** [comment]
Closing for now. I think we will want a Dockerfile somewhere at some point, so I'm leaving the branch.

**Pull request closed without merging**